cli/src/deploy/mod.rs

```diff
@@ -211,7 +212,7 @@ pub async fn get_config() -> HashMap<String, Vec<Config>> {
             "Credentials {} for deploy config {} not found in {}",
             cred_prof_name, deploy_profile_name, CREDENTIALS_FILE
           );
-          error!("INVALID_CREDENTIAL_ALIAS", "{}", err).await;
+          error!(ErrorKind::InvalidCredentialAlias as u8, "{}", err).await;
```

Comment:
Any way we can avoid the cast every time?

Reply:
Based on what I looked for, no. @cdmistman do you know if it's possible?

Reply:
why don't we just accept the `ErrorKind` in `client_error` and then cast it to `u8`?

Reply:
```rs
pub async fn client_error(err_code: ErrorKind, message: &str) {
  let mut body = json!({
    "errorCode": err_code as u8,
    ...
```

Reply:
That's actually the smart way to do it, thanks!